fiwareskuld/phase/phase0b_notify_users.py

```diff
@@ -90,22 +90,40 @@ def notify_users(self, user_ids):
         with open(filename, 'wf') as f:
             pickle.dump(user_ids, f, protocol=-1)
 
+    def _get_list(self, file):
+        try:
+            users = open(file)
+        except Exception:
+            logger.error('The users_to_notify.txt file must exists')
+            sys.exit(-1)
+
+        list_users = list()
+        for line in users.readlines():
+            user_id = line.strip()
+            if user_id == '':
+                continue
+            list_users.append(user_id)
+
+    def notify_trial_users(self):
+        """
+        It notify the trial users that they are going to be expired.
+        :return: nothing
+        """
+        list_trial_users = self._get_list('trial_users_to_notify.txt')
+        if list_trial_users:
+            self.notify_users(list_trial_users)
+
+    def notify_community_users(self):
+        """
+        It notify the community users that they are going to be expired.
+        :return: nothing
+        """
+        list_community_users = self._get_list('community_users_to_notify.txt')
+        if list_community_users:
+            self.notify_users(list_community_users)
 
 if __name__ == '__main__':
     logger = log.init_logs('phase0c')
```

Comment:
Something strange here, if the script is called phase0b why are you writing phase0c in the logs?